config/default.json

```diff
@@ -0,0 +1,12 @@
+{
+  "log": {
+    "level": "info"
+  },
+  "morello_host": {
+    "addres": "0.0.0.0",
```

Comment:
address

Reply:
we don't have one yet so it's all 0.0.0.0, happy to put loopback, ideas?

Reply:
As in the spelling of the key should be "address"

Reply:
Actually updated with '...' and instructions do say that this is something that will need to be updated

Reply:
It has been updated to 

```js
  address: '0.0.0.0;'
```